faker/providers/color/fr_FR/__init__.py

```diff
@@ -0,0 +1,156 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from collections import OrderedDict
+
+from .. import Provider as ColorProvider
+
+localized = True
```

Comment:
the `localized` module attribute is not needed anymore. You can delete this line